api/controllers/build.js

```diff
@@ -42,14 +42,23 @@ module.exports = {
   },
 
   create: (req, res) => {
```

Comment:
should we rename this `restart` or `createCopy` or something to make it more obvious?

Reply:
And/or even just a documentation comment would be good.

Reply:
I'll do both, naming it `create` doesn't necessarily fit, but I'd want the next developer to know why there isn't a `create` action.